tmux.conf

```diff
@@ -63,6 +63,5 @@ bind ^T split-window -h -p 20
 
 # Allow scrolling, pane/window selection and pane resizing with the mouse
 set -g mouse on
-set -g mouse-utf8 on
```

Comment:
tmate is a fork of an older tmux version, and this variable was added in a newer (post-fork) tmux version, so trying to set this var to any value results in an error